server/internal/server/org.go

```diff
@@ -85,6 +86,39 @@ func (s *S) AddUserToOrganization(ctx context.Context, req *v1.AddUserToOrganiza
 	return &v1.AddUserToOrganizationResponse{}, nil
 }
 
+// CreateDefaultOrganization creates the default org.
+// TODO(kenji): This is not the best place for this function as there is nothing related to
+// the server itself.
+func (s *S) CreateDefaultOrganization(ctx context.Context, c *config.DefaultOrganizationConfig) error {
+	_, err := s.store.GetOrganizationByTenantIDAndTitle(fakeTenantID, c.Title)
+	if err == nil {
+		// Do nothing.
+		return nil
+	}
+
+	if !errors.Is(err, gorm.ErrRecordNotFound) {
+		return err
+	}
+
+	org, err := s.CreateOrganization(ctx, &v1.CreateOrganizationRequest{Title: c.Title})
+	if err != nil {
+		return err
+	}
+
+	for _, uid := range c.UserIDs {
+		if _, err := s.AddUserToOrganization(ctx, &v1.AddUserToOrganizationRequest{
+			User: &v1.OrganizationUser{
+				OrganizationId: org.Id,
```

Comment:
```suggestion
				OrganizationId: org.OrganizationID,
```

Reply:
sorry ignore this

Reply:
Ah, I see. Yeah, this was confusing.. The code here is a bit weird as it's operating protos and RPC methods unnecessarily.